contracts/FungibleTokenSwitchboard.cdc

```diff
@@ -78,9 +80,6 @@ access(all) contract FungibleTokenSwitchboard {
                 emit VaultCapabilityAdded(type: vaultRef.getType(),
                                                switchboardOwner: self.owner?.address, 
                                                  capabilityOwner: capability.address)
-            } else {
-                // If there was already a capability for that token, panic
-                panic("There is already a vault in the Switchboard for this token")
```

Comment:
In my mind, this is a business logic change - was this intended. I don't necessarily see a problem with it, just making sure it was an intentional change.

Reply:
thats a good point. I just figured it made more sense to do a no-op here instead of panicing, but that might not be a good idea to do a business logic change like this in this PR. I'll add it back